.github/workflows/build.yml

```diff
@@ -501,17 +497,22 @@ jobs:
         working-directory: ./src/monarch
 
   test-bicep:
-    name: Test Bicep (${{ matrix.rid }})
-    runs-on: ${{ matrix.os }}
+    name: 'Run: ${{matrix.config.name}} (${{ matrix.runtime.rid }})'
```

Comment:
>: [](http://example.com/codeflow?start=14&length=1)

nit - None of the other job names use a `:` in the display name. I'd drop it or add it to all others.